### DISCUSSION TITLE
MHI2Q - Question about version and MIB

### DISCUSSION THREAD

**Discussion by Stick424** [Mr-MIBonk/M.I.B._More-Incredible-Bash#140] [Q&A]
Hi everyone, I bought a 2019 audi a4 and I want to upgrade the mapping so I have been looking into the possibilities. 

First of all thank you for all the work you are doing, I have read the different tutorials, downloaded MIB as well as the patches, but there is only one doubt now, it is in relation to my version of the MHI2Q, I wonder if I have to update it before installing MIB and change the FEC? 

Or can I do it as is? Thank you in advance for your answer, I put below the pictures I took.

![IMG_20230304_174619_HDR](https://user-images.githubusercontent.com/127192695/223421154-5a6570bc-d531-4ead-93e0-20025a0c7689.jpg)

![IMG_20230304_174631_HDR](https://user-images.githubusercontent.com/127192695/223421149-f87587f1-f96e-4e2a-900b-d369fca47f83.jpg)

**Comment by harman-f** [comment]
Hi, that version is fine.

**Reply by Stick424** [reply]
Ok thank you for your quick answer ! So i can install MIB following the procedure [https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/Prepare-M.I.B-SD-card-&-install-M.I.B#0---start-reading-here-if-you-are-completely-new-to-the-topic](here) and then concerning the FEC, should I click onthe AIO or "Flash patch image / add Fec " ?

**Comment by harman-f** [comment]
Yes, AIO

**Reply by Stick424** [reply]
Thanks !

**Reply by Stick424** [reply]
Hey Harman, so it worked fine, thanks for everything ! Gonna post on the thread about Navignore as it didnt worked for me this time !

**Reply by harman-f** [reply]
Did you use this M.I.B version?
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/archive/refs/heads/main.zip

**Reply by Stick424** [reply]
Hey Harman, no, I used another one as you told me in the other topic. Should I uninstall the old one and install the new one, or I can just put it in SD Card and it will work with new version ?

**Reply by harman-f** [reply]
You can install over
